tests/chainer_tests/functions_tests/math_tests/test_exponential_m1.py

```diff
@@ -0,0 +1,103 @@
+import unittest
+
+import numpy
+
+import chainer
+from chainer import cuda
+import chainer.functions as F
+from chainer import gradient_check
+from chainer import testing
+from chainer.testing import attr
+from chainer.testing import condition
+
+
+class UnaryFunctionsTestBase(object):
+
+    def make_data(self):
+        raise NotImplementedError
+
+    def setUp(self):
+        self.x, self.gy = self.make_data()
+
+    def check_forward(self, op, op_np, x_data):
+        x = chainer.Variable(x_data)
+        y = op(x)
+        gradient_check.assert_allclose(
+            op_np(self.x), y.data, atol=1e-7, rtol=1e-7)
+
+    def check_forward_cpu(self, op, op_np):
+        self.check_forward(op, op_np, self.x)
+
+    def check_forward_gpu(self, op, op_np):
+        self.check_forward(op, op_np, cuda.to_gpu(self.x))
+
+    @condition.retry(3)
+    def test_expm1_forward_cpu(self):
```

Comment:
I create a test_exponential_m1.py and a test_logarithm_p1.py, which I hope are tested parameterized correctly.